scipy/special/_cdflib.pyx

```diff
@@ -1273,7 +1273,7 @@ cdef inline (double, int, double) cdfbet_which3(
         dinvr(&DS, &DZ)
 
     if DS.status == -1:
-        return (DS.x, 1 if DS.qleft else 2, 0. if DS.qleft else INFINITY)
+        return (DS.x, 1 if DS.qleft else 2, 0. if DS.qleft else 1e100)
```

Comment:
For this and the others, can you remind why we're changing to match the upper bound from the original F77 but not the lower bound?

https://github.com/scipy/scipy/blob/1d3a067c2ccd0a6efddeb3194163aa9a3879d26e/scipy/special/cdflib/cdfbet.f#L280-L287

https://github.com/scipy/scipy/blob/1d3a067c2ccd0a6efddeb3194163aa9a3879d26e/scipy/special/cdflib/cdfbet.f#L108

But I saw for `cdftnc_which2` you are matching the lower bound as well.

Reply:
Because someone thought `1.0D-100` should be called zero so a not-so-smart dude in the future doesn't know how to read properly can make mistakes like this continuously.